packages/codegen-ui/lib/types/form/form-definition.ts

```diff
@@ -24,7 +25,7 @@ export type FormDefinition = {
     layoutStyle: StudioFormStyle;
   };
   elements: { [element: string]: FormDefinitionElement };
-  buttons: { [key: string]: string };
+  buttons: StudioFormCTAConfig;
```

Comment:
if you make this shape different from StudioFormCTAConfig, you can make it a lot more digestible by the renderer. For example, mapping the positions of the buttons in relation to each other, making label a required field, adding variants